api/src/wfl/references.clj

```diff
@@ -17,7 +17,7 @@
         :ref_sa          (prefix ".fasta.64.sa")}))
   ([]
    (let [prefix "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38"]
-     (reference_fasta prefix))))
+     (reference-fasta prefix))))
```

Comment:
Not sure this is an improvement, especially if the input is named `reference_fastas`.